<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>812</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget_canva" native="true">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>460</y>
     <width>781</width>
     <height>341</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btngraficar">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>100</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Entrenar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>20</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Regresion Lineal Neuronal</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Datos de entrenamiento</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>150</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Datos de prueba</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetentrenamiento">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>200</y>
     <width>281</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidgetprueba">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>200</y>
     <width>301</width>
     <height>191</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Iteraciones:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbxiteraciones">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
